CONTRIBUTING.md

```diff
@@ -36,49 +36,41 @@ the latest information.
 
 #### 1. Set up your Go environment
 
-This project is written in Go. To be able to contribute you will need a working Go installation. You can check the 
-[official installation guide](https://golang.org/doc/install). Make sure your `GOPATH` and `GOBIN` 
-[environment variables](https://github.com/golang/go/wiki/SettingGOPATH) are set correctly.
+This project is written in Go. To be able to contribute you will need:
 
-Here are some quick start steps you can follow:
+1. A working Go installation of Go >= 1.12. You can check the
+[official installation guide](https://golang.org/doc/install).
 
-Select a folder path for your go projects, for example `~/eksctl`
-```bash
-mkdir ~/eksctl
-cd ~/eksctl
-export GOPATH="$(pwd)"
-export GOBIN="${GOPATH}/bin"
-```
-
-> NOTE: Windows users should install Docker for Windows and run `make eksctl-image` to build their code.
+2. Make sure that `$(go env GOPATH)/bin` is in your shell's `PATH`. You can do so by
```

Comment:
Why?

Reply:
I recall this was needed during the build. Let me double-check.

Reply:
`mockery` wasn't found otherwise, but I think I can fix it so that it looks for mockery in the right place.

Reply:
Fixed